src/web/components/CMP.tsx

```diff
@@ -1,9 +1,14 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, Suspense } from 'react';
 import {
     shouldShow,
     setErrorHandler,
 } from '@guardian/consent-management-platform';
-import { ConsentManagementPlatform } from '@guardian/consent-management-platform/lib/ConsentManagementPlatform';
+
+const ConsentManagementPlatform = React.lazy(() =>
+    import(
+        /* webpackChunkName: "ConsentManagementPlatform" */ '@guardian/consent-management-platform/lib/ConsentManagementPlatform'
```

Comment:
`webpackChunkName` allows us to specify the name of the component, else webpack would default it to some random key, making hard to read